.dockerignore

```diff
@@ -164,8 +164,8 @@ scripts/oncall-email/today.json
 scripts/oncall-email/yesterday.json
 
 # Do not check in compiled binaries
-semgrep/semgrep/bin
-!semgrep/semgrep/bin/__init__.py
```

Comment:
with the entire `semgrep/semgrep/bin` dir being ignored, this line was a no-op

we must've force added the file originally, cause git wouldn't add it otherwise